### PR TITLE
add flowchart v1.4.1 with git auto-update, close #5056

### LINKED ISSUE #5056: [Request] flowchart.js
Surprised it is not in here !? 
https://github.com/adrai/flowchart.js
http://flowchart.js.org/

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/adrai/flowchart.js
**Watch 58
  Star 853
 Fork 128**
@Piicksarn could u help me check this PR for #5056? thank you!
because the `.js` include the version number, i didn't add auto-update config, and i have open the issue to ask the author if he can remove the version number.
- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

**Comment by Piicksarn** [comment]
@maruilian11, we should not put several unrelated commits in a same PR.
And in the second commit, the filename should not include version number.
The first commit is fine!

**Comment by maruilian11** [comment]
@Piicksarn i have corrected it~ could you take a look again? thank you!

**Comment by Piicksarn** [comment]
@PeterDaveHello,would you please take a look at the pr?thanks!

**Comment by PeterDaveHello** [comment]
@Piicksarn assign it to me next time.

We should take time to read its license. Pending.

**Comment by PeterDaveHello** [comment]
Assign to me next time @Piicksarn ...

**Comment by PeterDaveHello** [comment]
@cdnjs/intern please try to make sure its license is fine for us to add this lib on cdnjs.

**Comment by maruilian11** [comment]
@Piicksarn the author has change the license to `MIT`, i think it's fine for us to add it, can you check it again? thank you!

**Comment by Piicksarn** [comment]
@maruilian11, sorry for my late :sob: .....
i think it's fine but needed rebase to master!

**Comment by maruilian11** [comment]
@Piicksarn it's ok~ i know everyone in our team is busy.
could you help me check this PR again? thank you!

**Comment by Piicksarn** [comment]
@PeterDaveHello, would you mind to take a look at the pr? Thank you!

File: ajax/libs/flowchart/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "author": "adrai",
+  "name": "flowchart",
+  "version": "1.4.1",
+  "filename": "flowchart.min.js",
+  "private": false,
```
Comment (PeterDaveHello) [inline ajax/libs/flowchart/package.json]: We don't need this field.

**Comment by maruilian11** [comment]
@Piicksarn i have removed `private` field, could you help me check the PR again? thank you very much!

**Comment by PeterDaveHello** [comment]
and you didn't rebase ...

**Comment by maruilian11** [comment]
@Piicksarn could you help me check the PR again?

**Comment by Piicksarn** [comment]
@PeterDaveHello, would you have a look at the pr?
Thank you!

**Pull request merged**